docs/use-cases-periodic.md

```diff
@@ -1,18 +1,47 @@
 ---
 id: use-cases-periodic
-title: Periodic Execution (aka Distributed Cron)
+title: Periodic Execution
```

Comment:
This section seems very similar to monitoring right?

Reply:
I think you might be able to merge this in the other section. Not sure.

Reply:
Yes, actually I excluded it from the navigation pane and left the file to keep the existing links working. Maybe we should place permanent redirects instead.

Reply:
That works for me.

Reply:
Can you handle the redirect or should I have Ruslan help?

Reply:
I tried the [client redirects plugin](https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-client-redirects):

1. It wants me to delete the old page (okay)
2. It doesn't seem to play nicely with versioning: at least, I have to specify `/next` in the URLs, or otherwise it doesn't recognize the URL because `/docs/use-cases-monitoring` is not in the current version.
3. `yarn start` doesn't seem to respect the redirect, while `yarn build` does.

Should I go forward with this or does Ruslan have experience with redirects in docusaurus? Or maybe merge as-is and add redirects when the version is current?

Reply:
Ok so we've spent a lot of time (surprisingly) discussing this. Our decision is to not maintain backwards compatibility and blow away the old versions. That means you don't need to worry about redirecting, just update to the state you want for the currents docs.

:)